client/Assets/Scripts/AimDirection.cs

```diff
@@ -115,6 +112,33 @@ public void SetConeIndicator()
         cone.GetComponent<MeshFilter>().mesh = mesh;
     }
 
+    private Vector3 GetBisectorDirection()
+    {
+        Mesh coneMesh = cone.GetComponent<MeshFilter>().mesh;
+        Vector3[] coneVertices = coneMesh.vertices;
+
+        // Calculate the vertices of the triangle
+        Vector3 vertexA = cone.transform.TransformPoint(coneVertices[0]);
+        Vector3 vertexB = cone.transform.TransformPoint(coneVertices[1]);
+        Vector3 vertexC = cone.transform.TransformPoint(coneVertices[coneVertices.Length - 2]); // That is the last vertex
+
+        // Calculate the sides of the triangle
+        Vector3 sideAB = vertexB - vertexA;
+        Vector3 sideAC = vertexC - vertexA;
+
+        // return the bisector direction of the triangle's vertex angle
+        return (sideAB.normalized + sideAC.normalized).normalized;
+    }
+
+    public bool isInsideCone(GameObject player)
```

Comment:
```suggestion
    public bool IsInsideCone(GameObject player)
```
functions in c# should start with capital letter, after the change is done please check where it's used